content/html/concepts/attributes/style/style.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'Style'
+Description: 'The style attribute specifies how a page will be rendered in a web browser.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Style' 
+  - 'Attributes'
+  - 'Elements'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `style` attribute specifies how an element will be rendered in a web browser, e.g. the color or the size of a font in a paragraph.
+
+When we use the `style` attribute inline, it overrides any styles set globally. 
+
+**Note:** We can also specify the style of our elements through an external CSS style sheet or embedded in the `<head>` element using the `<style>` tag.
```

Comment:
```suggestion
> **Note:** We can also specify the style of our elements through an external CSS style sheet or embedded in the `<head>` element using the `<style>` tag.
```

Reply:
Let's refrain from using any pronouns like you and we.

Reply:
Done